raw_data/preprocessed/africa/sierra_leone_english.txt

```diff
@@ -0,0 +1,132 @@
+(1) In this Act, unless the context otherwise requires: "air" includes air within a building, a vehicle or any enclosure or structure or the airspace of Sierra Leone; "airspace" means the airspace within the territorial limits of Sierra Leone; "Director" means the Director for the Department of the Environment of the Ministry; "discharge" includes deposit, emission or leakage; "disposal" includes both land-based disposal and dumping in waters and airspace of Sierra Leone; "effluent limitation" means any restriction established by the Ministry on quantities, rates and concentration of chemical, physical, biological or other constituents which are discharged from point sources or other sources into the waters of Sierra Leone or on, or into the land; "environment" includes land, air, water and all plants, animals and human beings, living therein and the inter-relationship which exists among these or any of them; "licence" means environmental impact assessment licence; "Minister" means the Minister charged with responsibility for the environment; "Ministry" means the Ministry charged with responsibility for the environment; "monitoring" includes the inspection, measurement, sampling or analysis of the discharge of a pollutant or of any environmental medium in any locality, whether periodically or continuously;  an "offshore facility" means any facility of any kind located over, in, on, or under any of the waters of Sierra Leone; "onshore facility" means any facility of any kind (including motor vehicles and rolling stock) located over, in or under any land within Sierra Leone other than submerged land; "owner" includes: [a] in the case of a vessel the owner, operator or charterer of the vessel; [b] in the case of an onshore facility or an offshore facility, the owner or operator of such facility; [c] in the case of any abandoned offshore facility, the owner or operator of such facility immediately before its abandonment; and [d] in the case of a project, a person managing or controlling such project; "pollution" means any natural, man-made or man-aided alteration of the chemical, physical or biological quality of the environment to the extent that is detrimental to the environment or beyond acceptable limits and pollutants shall be construed accordingly; "prescribe" means prescribe by regulations; "project" means any activity which requires a licence under this Act; "removal" means removal of hazardous substances from the air, land or water, including shorelines or the taking of such other action as may be necessary to minimize damage to public welfare, ecology and natural resources of Sierra Leone; , "standards" includes criteria and specifications; "waters of Sierra Leone" means all water resources in any form, including atmospheric, surface and sub-surface and underground water-resources whether the water resources are in territorial waters, Exclusive Economic Zone or any area under the jurisdiction of the Republic of Sierra Leone. 
```

Comment:
Please remove the leading `an` and whitespace before `"offshore facility"`. Also, please remove the leading comma and whitespace before `"standards"`. Furthermore, please replace `water-resources` with `water resources`.